src/pages/transactions/TransactionForm.js

```diff
@@ -50,9 +51,10 @@ export default function TransactionForm({ onAddTransaction, onAlert }) {
     <div>
       <h1>Add Transaction</h1>
       <form onSubmit={handleSubmit}>
-        <button type="button" onClick={() => router.push("/")}>
-          Cancel
-        </button>
+        <Button
+          variant="secondary"
+          type="button"
+          onClick={() => router.push("/")}>Cancel</Button>
```

Comment:
Your button looks good, but I think you could better use a `Link` instead of the `button` if you just need to go back to the home page. It would be more semantic.

Reply:
Since it's a form, a cancel button is more userfriendly to exit the current action